OpenAPI/responses/201Created.json

```diff
@@ -0,0 +1,17 @@
+{
+    "description": "Created",
+    "headers" : {
+        "X-Location" : {
```

Comment:
Location is a standard HTTP header `X-` prefix not needed
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Location